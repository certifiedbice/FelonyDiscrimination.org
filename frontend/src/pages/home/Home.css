.home-h2,.home-h3 {
    text-align: center;
}