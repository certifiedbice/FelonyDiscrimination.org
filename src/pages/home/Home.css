main h2 {
    margin: 0 0 5px 0;
    font-weight: 700;
}
main p {
    font-size: 12px;
    font-weight: normal;
    line-height: normal;
}