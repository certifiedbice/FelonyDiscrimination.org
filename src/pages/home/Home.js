import React from 'react';
import './Home.css';
export default function Home(){
	return(
		<div>
			<h2>Our Mission:</h2>
			<p>
				Combating the rampant discrimination against persons with a felony history.
				If you have been the victim of this type of blanket policy, we invite you 
				to report that organization here. Opposite that, if you have been given an 
				opportunity, we invite you to recommend that organization here.
			</p>
		</div>
	);
}
