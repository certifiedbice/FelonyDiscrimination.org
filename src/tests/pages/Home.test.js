import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Home from '../../pages/home/Home';

it('renders without crashing',()=>{
    const div=document.createElement('div');
    ReactDOM.render(<Home/>,div);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<Home/>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});