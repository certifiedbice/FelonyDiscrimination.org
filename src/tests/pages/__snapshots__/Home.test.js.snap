// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Our Mission:
  </h2>
  <p>
    Combating the rampant discrimination against persons with a felony history. If you have been the victim of this type of blanket policy, we invite you to report that organization here. Opposite that, if you have been given an opportunity, we invite you to recommend that organization here.
  </p>
</div>
`;
